confluent_kafka/src/confluent_kafka.c

```diff
@@ -1629,6 +1629,18 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
                 PyDict_Update(confdict, kwargs);
         }
 
+        if (ktype == RD_KAFKA_CONSUMER) {
+                ko = cfl_PyUnistr(_FromString("group.id"));
+                if (PyDict_Contains(confdict, ko) != 1) {
```

Comment:
Using `PyDict_GetItemString()` avoids the extra object ko object creation.

Reply:
good point, I see now it returns a borrowed reference; stand-by I'll fix that I just though `_Contains` felt more propper.